### PR TITLE
readme

### PR DESCRIPTION
**Pull request by TrunkOfUkuleles**
[ x ] Top-level README “Table of Contents” is updated
[ x ] Feature tasks for this challenge are completed
[ x ] Unit tests written and passing
[ x ] “Happy Path” - Expected outcome
[ x ] Expected failure
Edge Case (if applicable/obvious)
[ x ] README for this challenge is complete
[ x ] Summary, Description, Approach & Efficiency, Solution
[ x ] Link to code
[ x ] Picture of whiteboard

### CODE REVIEW

File: javascript/data-structures/tree/__tests__/tree.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+
+const Node = require('../node.js')
+const { BinaryTree , BinarySearchTree } = require('../tree.js');
+
+describe("Stacks and Queues Testing", () => {
```
Comment (lizkavalski) [inline javascript/data-structures/tree/__tests__/tree.test.js:7]: rename the test
